automation/swift-components-docs/replace-imports.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# To build a Swift project, we cannot import "MozillaRustComponents" from the modulemap.
+# For it to work, the module needs another name. This script replaces
+# import MozillaRustcomponents
+# with
+# import MyCMap
+# 'MyCMap` is the name of the module the `module.modulemap` is exporting.
```

Comment:
I'm don't totally understand this one, but could this be avoided if UniFFI had support for generating a modulemap for all the components?

Reply:
Yeah that's a weird one. At this point, I surrender to the Swift gods. I tried what feels like a hundred different setups until `spm` finally included the header files.

The issue, I believe, is, that the project is named `MozillaRustComponents`, and the `.swift` files try to import `MozillaRustComponents` as well, because that's the name of the generated `module.modulemap` file. It works for `Xcframework`, but it didn't work for this setup (pure Swift). 

I can play around with the namings and see if we can leave the imports as is...